tests/integration_tests/core/test_tracker_store.py

```diff
@@ -0,0 +1,139 @@
+import logging
+from unittest.mock import Mock
+
+import pytest
+from _pytest.logging import LogCaptureFixture
+from _pytest.monkeypatch import MonkeyPatch
+import sqlalchemy as sa
+
+from rasa.core.tracker_store import SQLTrackerStore
+from .conftest import (
+    POSTGRES_HOST,
+    POSTGRES_PORT,
+    POSTGRES_LOGIN_DB,
+    POSTGRES_TRACKER_STORE_DB,
+    POSTGRES_USER,
+    POSTGRES_PASSWORD,
+)
+
+# NOTE about the timeouts in this file. We want to fail fast
+# because SQLTrackerStore tries to connect several times
+# until it works. If the timeout is hit, it probably means
+# that something is wrong in the setup of the test
+
+
+@pytest.mark.sequential
+@pytest.mark.timeout(10)
+def test_sql_tracker_store_with_login_db(
+    postgres_login_db_connection: sa.engine.Connection,
+):
+    tracker_store = SQLTrackerStore(
+        dialect="postgresql",
+        host=POSTGRES_HOST,
+        port=POSTGRES_PORT,
+        username=POSTGRES_USER,
+        password=POSTGRES_PASSWORD,
+        db=POSTGRES_TRACKER_STORE_DB,
+        login_db=POSTGRES_LOGIN_DB,
+    )
+
+    matching_rows = (
+        postgres_login_db_connection.execution_options(isolation_level="AUTOCOMMIT")
+        .execute(
+            sa.text(
+                "SELECT 1 FROM pg_catalog.pg_database WHERE datname = :database_name"
```

Comment:
can we get this info from the sqlalchemy session?

Reply:
I'm not sure what you exactly mean here @joejuzl, can you detail a bit more? Are you talking about using `SQLTrackerStore.session_scope()`?

Reply:
I just wondered if it was possible to see what databases existed through an sqlalchemy method rather than having to query sql directly.

Reply:
🤔 I didn't find a way to do that with sqlalchemy